tests/e2e/common.go

```diff
@@ -156,32 +157,122 @@ func deleteVeleroCR(client dynamic.Interface, instanceName string, namespace str
 	return veleroClient.Delete(context.Background(), instanceName, metav1.DeleteOptions{})
 }
 
+// Keeping it for now
+func isNamespaceExists(namespace string) error {
```

Comment:
I would rename this. `isNamespaceExists` should be `doesNamespaceExist` and this should return a boolean. Ideally we would do:
```
func doesNamespaceExist(namespace string) (bool, error)
```